modules/telemetry/include/hephaestus/telemetry/telemetry.h

```diff
@@ -0,0 +1,47 @@
+//=================================================================================================
+// Copyright (C) 2023-2024 HEPHAESTUS Contributors
+//=================================================================================================
+
+#pragma once
+
+#include <type_traits>
+
+#include <absl/base/thread_annotations.h>
+
+#include "hephaestus/serdes/json.h"
+#include "hephaestus/telemetry/sink.h"
+
+namespace heph::telemetry {
+
+void registerSink(ITelemetrySink* sink);
+
+void metric(const MetricEntry& log_entry);
```

Comment:
nit: since its a function I would prefer a verb here.
how about
```suggestion
void measure(const MetricEntry& log_entry);
```

Reply:
@lhruby and @floriantschopp are the naming master so I will defer the decision to them.